src/lib/isMobilePhone.js

```diff
@@ -118,7 +118,7 @@ const phones = {
   'kk-KZ': /^(\+?7|8)?7\d{9}$/,
   'kl-GL': /^(\+?299)?\s?\d{2}\s?\d{2}\s?\d{2}$/,
   'ko-KR': /^((\+?82)[ \-]?)?0?1([0|1|6|7|8|9]{1})[ \-]?\d{3,4}[ \-]?\d{4}$/,
-  'ky-KG': /^(\+?7\s?\+?7|0)\s?\d{2}\s?\d{3}\s?\d{4}$/,
+  'ky-KG': /^(\+996\s?)?(22[0-9]|50[0-9]|55[0-9]|70[0-9]|75[0-9]|77[0-9]|880|990|995|996|997|998)\s?\d{3}\s?\d{3}$/,
```

Comment:
Upon quick comparison with the latest PDF of the ITU ( https://www.itu.int/oth/T0202000074/en ), I notice some missing mobile phone numbers. For example the relatively new +996 60 0 XXX XXX is missing, but also phone numbers from Table 3 (pages 7 and 8). Could you go through that document and update the regular expression accordingly?

Reply:
Yes, I can, but the information in the document is not up-to-date.

The numbers +996 60 0 XXX XXX are not used by the mobile operator beeline.kg (Sky Mobile LLC). You can check the list of all available codes here - https://nomer.beeline.kg/#/

As for the numbers on pages 7 and 8, the mobile operators (Aktel, Katel) no longer provide their services.
I found this information, but it's on Russian language.

[about KATEL](https://economist.kg/mnenie/2021/03/13/istoriya-katel-pervogo-sotovogo-operatora-kyrgyzstana/#:~:text=%D0%98%D0%B7%20%D0%BF%D1%80%D0%B5%D1%81%D1%81%2D%D1%80%D0%B5%D0%BB%D0%B8%D0%B7%D0%B0%20%D0%BE%D1%82%2024,%D0%BA%D0%BE%D0%BC%D0%BF%D0%B0%D0%BD%D0%B8%D1%8F%20%D0%BE%D1%81%D1%82%D0%B0%D0%B2%D0%B8%D0%BB%D0%B0%20%D0%BC%D0%BD%D0%BE%D0%B3%D0%BE%D0%BC%D0%B8%D0%BB%D0%BB%D0%B8%D0%BE%D0%BD%D0%BD%D1%8B%D0%B5%20%D0%BD%D0%B5%D1%83%D0%BF%D0%BB%D0%B0%D1%87%D0%B5%D0%BD%D0%BD%D1%8B%D0%B5%20%D0%B4%D0%BE%D0%BB%D0%B3%D0%B8.)
[about Aktel](https://www.vb.kg/doc/256308_novym_rykovoditelem_fonex_stal_dair_tuleev.html)

Reply:
Certainly, here is the additional information:

+996 56X XXXXXX - Mobile - Winline (Sapatcom) - The operator is not operational.
+996 57X XXXXXX - Mobile - Sotel (Nexi) - The operator is not operational.

Reply:
These companies went bankrupt and closed even before the pandemic. : )

Reply:
@WikiRik hello!
I wanted to follow up on my previous comment from 5 days ago. I'm looking forward to hearing your response. If you have any questions or need further clarification, please feel free to ask.

Thank you in advance for your attention.

Reply:
Is anyone here? )